src/app/content/practiceQuestions/components/Answer/index.tsx

```diff
@@ -63,19 +72,22 @@ const Answer = ({
       id='i18n:practice-questions:popup:answers:choice'
       values={{choiceIndicator: choiceIndicator.toUpperCase()}}
     >{(msg: string) =>
-      <AnswerLabel aria-label={msg}>
-        {choiceIndicator}
+      <React.Fragment>
         <input
+          id={choiceIndicator}
           type='radio'
           name={choiceIndicator}
           checked={isSelected}
           disabled={isSubmitted}
           onChange={onSelect}
         />
-      </AnswerLabel>
+        <AnswerLabel ref={answerRef} aria-label={msg} for={choiceIndicator} tabIndex={-1}>
```

Comment:
when karina did this she used [htmlFor](https://reactjs.org/docs/dom-elements.html#htmlfor)